src/app/api/manuscript.api.ts

```diff
@@ -78,18 +77,25 @@ export async function getManuscriptContent(id: string): Promise<Manuscript> {
 
 export function syncChanges(id: string, changes: ManuscriptDiff[]): Promise<void> {
   // TODO: squash changes here
-  return axios.put(manuscriptUrl(id), changes.map(makeChangesSeralizable));
+  const backendTranscations = changes.map(makeChangesSeralizable).filter((transcation) => transcation !== null);
+  return axios.post(manuscriptUrl(id), { changes: backendTranscations });
```

Comment:
are we not following REST then?

Reply:
Well, I think a post is fine in this context as it's creating a resource rather than updating any. It's creating a new change resource that is yet to be applied.

Reply:
`POST` seems correct here but this should be `  return axios.post(`${manuscriptUrl(id)}/changes`, { changes: backendTranscations });
`